### ISSUE TITLE
“clonal T cell receptor rearrangement”

### ISSUE THREAD

**Issue by MorganSimiluk** [obophenotype/human-phenotype-ontology#4655]
**Preferred term label:**
“clonal T cell receptor rearrangement”

**Synonyms**

**Definition (free text, please give PubMed ID)**
29055897:
"Modern diagnostic pathology has exploited the molecular interrogation of the TCR for a number of purposes: clonality may serve as evidence that an apparently malignant lymphoid process with indefinite immunophenotypic lineage specificity is likely of T cell origin; clonality may establish a form of molecular confirmation in instances in which the ‘malignity’ of a lymphoid lesion is not certain based solely on histopathological or immunophenotypic assessment; clonality may be used as a means of minimal residual disease (MRD) testing; and clonality forms reliable proof that two seemingly distinct T cell lymphoproliferative disorders are derived from the same malignant clonal population, and to provide molecular genetic evidence of a clonal T cell infiltrate accompanying another malignant process.3–6"

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by nicolevasilevsky** [comment]
I am not sure I understand this term request - this doesn't sound like a phenotype to me

**Comment by pnrobinson** [comment]
I think we mean the fact that there is a highly abundant T Cell clone (with the same TCR) in the circulation. I will work on the definition

**Comment by nicolevasilevsky** [comment]
Ah, thanks, @pnrobinson!

**Comment by pnrobinson** [comment]
@MorganSimiluk  This publication does not describe the outcomes of the test for T cell clonality. 
Are we interested in Monoclonality as an outcome (e.g. indicative of a cancer such as cutaneous T-cell lymphoma) or are we interested in the presence of an obvious clone (e.g., presence of a predominant band on PCR based assays?). What is the preferred terminology for reporting these findings?

Would something like this be correct?

Label: Clonal T cell receptor rearrangement
Def: Presence of a predominant T cell clone. In PCR-based assays, this finding is inferred on the basis of one or two prominent bands within a valid size range. In NGS-based assays, this finding is inferred on the basis of a high number of reads that map to a single T cell receptor clone.

**Comment by MorganSimiluk** [comment]
Based on my understanding of this issue, what you suggest is on target.

**Comment by pnrobinson** [comment]
Adding term as HP_0032577
